packages/stores/src/account/base.ts

```diff
@@ -792,6 +830,14 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
       signMode
     );
 
+    console.log(
+      "decode bytes",
+      wallet?.signingStargateOptions?.registry?.decodeTxBody(
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        signedTxBodyBytes!
+      )
+    );
```

Comment:
Here, it decodes to zero.

Therefore, something in the serialization logic is broken